Civi/Api4/Service/Spec/FieldSpec.php

```diff
@@ -412,49 +405,31 @@ public function setOutputFormatters($outputFormatters) {
    * @return $this
    */
   public function addOutputFormatter($outputFormatter) {
+    if (!$this->outputFormatters) {
+      $this->outputFormatters = [];
+    }
     $this->outputFormatters[] = $outputFormatter;
 
     return $this;
   }
 
-  /**
-   * @return bool
-   */
-  public function getreadonly() {
-    return $this->readonly;
-  }
-
   /**
    * @param bool $readonly
    * @return $this
    */
-  public function setreadonly($readonly) {
+  public function setReadonly($readonly) {
```

Comment:
Hmm - what about proper camels setReadOnly - is that inconsistent with other places?

Reply:
The property is called "readonly" (all lower case)

Reply:
:-(